src/modules/shared/components/Blade/Blade.module.scss

```diff
@@ -7,8 +7,8 @@
 	visibility: hidden;
 	display: grid;
 	grid-template-rows: auto auto 1fr auto;
-	width: 100%;
-	max-width: 46.4rem;
+	width: 46.4rem;
+	max-width: 92%;
```

Comment:
Ik vermijd liever procentuele breedtes, is er geen manier om `margin/padding` te gebruiken om een minimumafstand met de linker-edge te garanderen?